### ISSUE TITLE
network cost don't convergence

### ISSUE THREAD

**Issue by beimingmaster** [MorvanZhou/Reinforcement-learning-with-tensorflow#18]
RL_brain.py and DQN_modified.py 

network cost don't convergence

environments:
python3
tensorflow 1.3.0
macbookpro

**Comment by MorvanZhou** [comment]
In those scripts, if you want to converge, you have to lower the learning rate gradually.
